nodejs_container/index.js

```diff
@@ -4,52 +4,25 @@ const fs = require('fs');
 const path = require('path');
 
 // deals with ensuring the correct version for the machine/node version
-const binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
-
-const HolochainApp = require(binding_path).HolochainApp;
+const binding_path = binary.find(path.resolve(path.join(__dirname, './package.json')));
+
+const { ConfigBuilder, Habitat } = require(binding_path);
+
+Habitat.prototype.callRaw = Habitat.prototype.call
+Habitat.prototype.call = function (id, zome, trait, fn, params) {
```

Comment:
I'm playing around with what it feels like to call a Container a Habitat. We can change this before merge. (Or not :smiling_imp: )